crates/masonry/src/widget/flex.rs

```diff
@@ -482,12 +462,83 @@ impl<'a> FlexMut<'a> {
         Some(self.ctx.get_mut(child))
     }
 
+    pub fn update_child_flex_params(&mut self, idx: usize, params: impl Into<FlexParams>) {
+        let child = &mut self.widget.children[idx];
+        let child_val = std::mem::replace(child, Child::FixedSpacer(0.0, 0.0));
+        let widget = match child_val {
+            Child::Fixed { widget, .. } | Child::Flex { widget, .. } => widget,
+            _ => {
+                panic!("Can't update flex parameters of a spacer element");
+            }
+        };
+        let new_child = new_flex_child(params.into(), widget);
+        *child = new_child;
+        // TODO
```

Comment:
Are these `TODO` comments, a result of copy paste, or is there really something todo? (and what?)

Btw. I think the two lines below this, could be summed up with
```rust
        self.ctx.children_changed();
```
couldn't it?